src/components/entity/filter/EntityFilterFacetPanels.vue

```diff
@@ -0,0 +1,158 @@
+                    v-bind="attrs"
+                    v-on="on"
+                  ></v-text-field>
+                </div>
+              </template>
+              <v-date-picker
+                v-model="fromDate"
+                @input="menuFrom = false"
+                @change="handleMinDateChange"
+              ></v-date-picker>
+            </v-menu>
+          </v-col>
+
+          <v-col cols="12" sm="6" md="4">
+            <v-menu
+              v-model="menuTo"
+              :close-on-content-click="false"
+              :nudge-right="40"
+              transition="scale-transition"
+              offset-y
+              min-width="auto"
+            >
+              <template v-slot:activator="{ on, attrs }">
+                <div class="text-body-2 float-middle">
+                  <v-text-field
+                    v-model="toDate"
+                    label="To:"
+                    append-icon="mdi-calendar"
+                    readonly
+                    v-bind="attrs"
+                    v-on="on"
+                  ></v-text-field>
+                </div>
+              </template>
+              <v-date-picker
+                v-model="toDate"
+                @input="menuTo = false"
+                @change="handleMaxDateChange"
+              ></v-date-picker>
+            </v-menu>
+          </v-col>
+          <v-spacer></v-spacer>
+        </v-row>
+      </template>
+    </CustomFilterFacetPanel>
+
+    <CustomFilterFacetPanel>
+      <template #title>Show Expired</template>
+      <template #content>
+        <v-container>
+          <v-col>
+            <p>Show Expired <v-switch @change="handleSwitchChange" v-model="showExpired"></v-switch></p>
+          </v-col>
+        </v-container>
+      </template>
+    </CustomFilterFacetPanel>
+  </v-expansion-panels>
+</template>
+
+<script lang="ts">
+import {
+  IEntityFacetField,
+} from "@/interfaces/api/v2/entityFilter.interface";
+import { Component, Vue } from "vue-property-decorator";
+import { mapActions, mapGetters } from "vuex";
+import EntityFilterFacetPanel from "@/components/entity/filter/EntityFilterFacetPanel.vue";
+import CustomFilterFacetPanel from "@/components/entity/filter/CustomFilterFacetPanel.vue";
+import { Filter } from "@/store/modules/entityFilters";
+
+@Component({
+  components: {
+    EntityFilterFacetPanel,
+    CustomFilterFacetPanel,
+  },
+  computed: {
+    ...mapGetters([
+      "searchTopicFacets",
+      "getAuthorities",
+      "getCredentialTypes",
+      "getRegistrationTypes",
+      "getEntityFilters",
+    ]),
+  },
+  methods:{
+    ...mapActions([
+      "setFilter"
+    ]),
+  }
+})
+export default class EntityFilterFacetPanels extends Vue {
+  private getAuthorities!: Array<IEntityFacetField>;
+  private getCredentialTypes!: Array<IEntityFacetField>;
+  private getRegistrationTypes!: Array<IEntityFacetField>;
+  getEntityFilters!: Filter;
+  setFilter!:(filter:Filter) => void;
+  data() {
```

Comment:
Make sure to type the return values on methods otherwise the linter will complain

Reply:
change completed